WebRoot/css/zm.css

```diff
@@ -3757,7 +3792,11 @@ HTML>BODY .appt-selected .appt_allday_body
 
 /* Table for a section of prefs (head + body) */
 .ZOptionsSectionTable {
-	margin-bottom:1.5em;
+	@PrefSectionContentTable@
+}
+
+.ZOptionsSectionTable * {
+	@PrefViewFontSize@
```

Comment:
It's there, Large diffs are not rendered by default so skin.properties was not loaded